guides/source/caching_with_rails.md

```diff
@@ -41,27 +49,10 @@ around with caching locally by running `rails dev:cache`, or by setting
 NOTE: Changing the value of `config.action_controller.perform_caching` will
 only have an effect on the caching provided by Action Controller.
 For instance, it will not impact low-level caching, that we address
-[below](#low-level-caching).
+[below](#low-level-caching-using-rails-cache-fetch).
 
 [`config.action_controller.perform_caching`]: configuring.html#config-action-controller-perform-caching
 
-### Page Caching
```

Comment:
Since these features, Page Caching and Action Caching, have been deprecated since Rails 4, I thought we could remove this section and any references to it. Additionally, the [[actionpack-page_caching gem](https://github.com/rails/actionpack-page_caching)](https://github.com/rails/actionpack-page_caching) has not received updates in three years. 

I can revert these changes if we don't want to remove it.